example/example.c

```diff
@@ -139,13 +146,13 @@ int main(void) {
 
 #if !defined(SIR_NO_SYSTEM_LOGGERS)
```

Comment:
Wouldn't this decrease coverage? We want to hit those exports even with NO_SYSTEM_LOGGERS, yeah?

Reply:
That's unchanged from the original.  I didn't try to change that (yet).

Reply:
Yeah, we should remove that. Even though we hit those paths during testing, can't hurt and makes the .c look less terrible.